docs/sdk.rst

```diff
@@ -169,58 +169,73 @@ latter also works with the ``Executor`` class.
 Batching
 --------
 
-The SDK includes a batch interface to reduce the overheads of launching a function many times.
-To use this interface, you must first create a batch object and then pass that object
-to the ``batch_run`` function. ``batch_run`` is non-blocking and returns a list of task ids
-corresponding to the functions in the batch with the ordering preserved.
+The SDK includes a batch interface to reduce the overhead of launching a function many
+times.  To use this interface, first create a |Batch| object and then pass that
+object to the |batch_run()| method.  The |batch_run()| method only blocks for the
+duration of the API call; the tasks may not yet have been received by the endpoint when
+it returns.
 
 .. code-block:: python
 
+  def noisy_multiply(x, y, noise=1, offset=0) -> float:
+      import random
+      return x * y + random.uniform(offset - noise, offset + noise)
+
+  noisy_fn_id = gcc.register_function(noisy_multiply)
   batch = gcc.create_batch()
 
-  for x in range(0,5):
-    batch.add(x, endpoint_id=tutorial_endpoint, function_id=func_id)
+  for val in range(5):
+    batch.add(noisy_fn_id, val + 3, val)
 
-  # batch_run returns a list task ids
-  batch_res = gcc.batch_run(batch)
+  # See what data structure will be sent to the API with .prepare()
+  # print(batch.prepare())
```

Comment:
We shouldn't write an example here of the output from the print as that would need to be updated each time.  Wonder if it's useful to show at least the start of it?

Reply:
I'm less concerned with the specific `.prepare()` output at this juncture in the documentation.  I intended this comment to give users a hint as to how to gain some insight into the plumbing.  Sometimes, staring at the raw data is more helpful that reading documentation; the point is to leave a crumb here that folks can idly use later when debugging their code.